### ISSUE TITLE
Why do we make notebooks in the data/ directory?

### ISSUE THREAD

**Issue by marwahaha** [swcarpentry/python-novice-inflammation#449]
We just ran this workshop. It seemed clear to create a notebook in the python-novice-inflammation/ folder, but all the examples are listed with `inflammation-01.csv` (we got tons of `no file or directory found` errors; quick fix is `!pwd` in the notebook to see where you are, then probably change to `data/inflammation-01.csv` ). The [setup is at least consistent](http://swcarpentry.github.io/python-novice-inflammation/setup/) (i.e., create a Jupyter notebook in the data folder), but still feels odd.

At the very least, could we make the file paths absolute (and maybe fewer folders / a folder with a shorter name) ? This is a very common issue (happens every time I teach it), and causes a lot of helper/learner headache and pain early in the lesson.

**Comment by annefou** [comment]
That's true. Even when checking the setup instructions during the class (showing they should have 3 files "inflammation-01.csv, inflammation-02.csv, inflammation-03.csv"), we still have attendees getting "no file or directory found".  It's a good opportunity to refresh their memory (they usually had a bash shell lesson before or the day before) and also show them how to "navigate" when starting Jupyter notebook.  !pwd would not work for Windows users  (!dir should work). 
We can then focus on the python language for the rest of the lesson.
